bin/cpumask.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python3
+
+'''Convert CPU list(s) to CPU masks'''
+
+import argparse
+from dataclasses import dataclass
+
+
+@dataclass
```

Comment:
who are you, stranger? what you doing here, dataclass "decorator" ??

Reply:
I did learn some stuff from my pylink induced exercises with roadblock:

https://github.com/perftool-incubator/roadblock/commit/23ffc5b5a213ae0adc661520968bd00f2a665755

Reply:
hmm there are so many different ways to use these decorators...

Reply:
pylint didn't like me using a class as just a place to store data, it complained about a number of different things (contstructors, number of properties, etc.) and switching it to a dataclass made all that go away.